poppy/__init__.py

```diff
@@ -93,13 +93,15 @@ class Conf(_config.ConfigNamespace):
 from . import optics
 from . import misc
 from . import fresnel
+from . import wavefront
```

Comment:
let's consider renaming this `physical_wavefront.py` or similar. Plain `wavefront.py` makes it sound like it contains the definitions of the `Wavefront` class itself which it doesn't.